test/framework/flux.go

```diff
@@ -552,5 +582,5 @@ func (e *E2ETest) clusterConfigGitPath() string {
 }
 
 func RequiredFluxEnvVars() []string {
-	return requiredEnvVars
+	return append(fluxRequiredEnvVars, requiredEnvVars...)
```

Comment:
this could modify the original slice `fluxRequiredEnvVars`, so you might end up with duplicated elements if you run this method twice. I would advice against that

Reply:
Thanks for the input!
This actually got me to realize we only needed `fluxRequiredEnvVars ` over here and the others get set from when we do vsphere setup.